projects/batfish/src/org/batfish/grammar/cisco/Cisco_snmp.g4

```diff
@@ -103,6 +105,7 @@ ss_null
       | QUEUE_LENGTH
       | SOURCE_INTERFACE
       | TCP_SESSION
+      | TFTP_SERVER_LIST
```

Comment:
Since in your unit test you are using this to handle a line with a reference to an access-list, `TFTP_SERVER_LIST` should not appear in `ss_null` but rather be handled. Otherwise we may have false positives for unreferenced access-lists.